tests/unit/work-items/work-items.test.ts

```
@@ -82,6 +82,38 @@ describe(!!(config.workspaceSlug && config.projectId && config.userId), "Work It
+
+    try {
+      pqlWorkItem = await client.workItems.create(workspaceSlug, projectId, {
+        name,
+        priority: "high",
+      });
+
+      const filtered = await client.workItems.list(workspaceSlug, projectId, {
+        pql: 'priority IN ("high")',
+      });
+
+      expect(filtered).toBeDefined();
```

ESLint

[error] 'expect' is not defined.

(no-undef)